bitcoin/src/blockdata/weight.rs

```diff
@@ -88,7 +88,7 @@ impl Weight {
 
     /// Checked subtraction.
     ///
-    /// Computes `self - rhs` returning `None` if an overflow occurred.
+    /// Computes `self - rhs` returning `None` if an underflow occurred.
```

Comment:
This change is unneeded, the term "overflow" can be applied in both directions.  I know it reads strangely but I've looked this up multiple times. The change is harmless but I used "overflow" in another PR recently.

Reply:
Using underflow when subtraction happens is consistent with other rust projects from what I've seen, for example see: https://docs.rs/num/latest/num/trait.CheckedSub.html.  It doesn't say overflow, it describes underflow.  I think because you used overflow in another PR is not really a reason to just call everything an overflow.

Reply:
I think it's OK to overload the term overflow in obvious cases like subtraction but underflow is more readable name anyway, so it's strictly better. I don't think it's super-important though.

Reply:
I don't think this was a waste of time.  Perhaps it's an oversight in the rust-docs.  I contribute to the rust lang docs as well and maybe I'll put in a PR to fix this with rust docs.  I'm glad you pointed that out instead of ignoring me!

Reply:
@tcharding you're correct, underflow is the right term either way it wraps around.  Underflow is reserved for cases where there's not enough digits to represent a very small number near zero https://en.wikipedia.org/wiki/Arithmetic_underflow.  My mistake, I should have spent some more time researching this instead of looking at another crate and assuming I understood the meaning.

Reply:
Thanks for the link, I did not know that underflow was reserved for floating points. TIL. Thanks also for admitting the mistake.